services/server/config/config.go

```diff
@@ -80,6 +80,8 @@ type StreamProcessor struct {
 	Path string `toml:"path"`
 	// Args to the binary
 	Args []string `toml:"args"`
+	// Environment variables for the binary
+	Envs []string `toml:"envs"`
```

Comment:
Nit: Envs -> Env.

In my understanding "an environment" means a set of variables, not a single variable

Reply:
I now renamed all occurrences of 'Envs' and 'envs' and removed the trailing 's'.

Reply:
`Envs` -> `Env` (so field name corresponds to toml)

Reply:
oooh, I though I had replaced them all. Thanks!